docs/uml/protocol/ExtEvSimulationSequence.puml

```diff
@@ -1,10 +1,46 @@
 @startuml
-SimScheduler -> ExtSimAdapter: <font color=red>!</font> <font color=green>ActivityStartTrigger</font>
+
+!theme plain
+
+==Init==
+SimScheduler -> ExtSimAdapter: <font color=red>!</font> <font color=green>ActivityStartTrigger(-1L)</font>
+activate ExtSimAdapter
+
+ExtSimAdapter -> ExtSimulation: queue(ActivityStartTrigger)
+deactivate ExtSimAdapter
+activate ExtSimulation
+note right: Initialize external\nmobility simulation
+
+ExtSimulation -> ExtSimAdapter: <font color=red>!</font> CompletionMessage(newTriggers)
+deactivate ExtSimulation
+activate ExtSimAdapter
+
+ExtSimAdapter -> SimScheduler: <font color=red>!</font> <font color=green>CompletionMessage(newTriggers)</font>
+deactivate ExtSimAdapter
+
+==Sim==
+SimScheduler -> ExtSimAdapter: <font color=red>!</font> <font color=green>ActivityStartTrigger(tick)</font>
 activate ExtSimAdapter
 
 ExtSimAdapter -> ExtSimulation: queue(ActivityStartTrigger)
 deactivate ExtSimAdapter
 activate ExtSimulation
+
+ExtSimulation -> ExtEvDataService: <font color=red>!</font> RequestEvcsFreeLots
+activate ExtEvDataService
+ExtEvDataService -> EvcsAgent1: <font color=red>!</font> EvFreeLotsRequest(tick)
+activate EvcsAgent1
+ExtEvDataService -> EvcsAgent2: <font color=red>!</font> EvFreeLotsRequest(tick)
+activate EvcsAgent2
+EvcsAgent2 -> ExtEvDataService: <font color=red>!</font> FreeLotsResponse(_, _)
+deactivate EvcsAgent2
+EvcsAgent1 -> ExtEvDataService: <font color=red>!</font> FreeLotsResponse(_, _)
+deactivate EvcsAgent1
+ExtEvDataService -> ExtSimulation: <font color=red>!</font> ProvideEvcsFreeLots(_)
+deactivate ExtEvDataService
+
+'TODO Refactor from here
```

Comment:
Is there still refactoring needed? Maybe, @speter, could you once again verify, that this protocol reflects the truth? Somewhat the handling of the `EvMovementsMessage` feels a bit clumpsy...

Reply:
I just realized that you added this todo. I checked the diagram again, and it seems all in place (besides some minor syntax questions). Can you recall what needs to be done here?

The only small problem I found is that there's now an inconsistency between sometimes specifying message parameters (e.g. `ActivityStartTrigger(tick)`) and sometimes not (e.g. `EvMovementsMessage`). To me, it is still unclear how helpful it is to use placeholder parameters (e.g. in `FreeLotsResponse(_, _)`) in contrast to not specifying parameters at all.

Reply:
Oh, sorry! My bad! Didn't remeber, that I also had hands on it. Most likely, I just forget to remove the comment. And thanks a lot for reconfirming!

Oftentimes, I left out the parameters, when there were too many. But I appreciate your proposal with placeholder arguments! So, feel free to add them or not.

Reply:
There actually _was_ some stuff missing, I adapted the diagram so that it should now reflect the implemented protocol. Also added parameters where it made sense to me.